CancerGenomicsExamples.fsh

```diff
@@ -35,3 +36,24 @@ InstanceOf: GenomicRegionStudied
 // Commented out -- causes SUSHI to not create this example -- see issue https://github.com/FHIR/sushi/issues/112
 //* component[GeneStudied].valueCodeableConcept = HGNC#619728 "STK11" 
 
+Instance: mCODETumorMarkerExample01 
+InstanceOf: TumorMarker
+* id = "mCODETumorMarkerExample01"
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TumorMarker"
+* status = #final "final"
+* code = LNC#39004-7 "Epidermal growth factor receptor Ag [Presence] in Tissue"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* performer = Reference(mCODEPractitionerExample01) 
+* valueCodeableConcept = SCT#10828004 "Positive (qualifier value)"
+
+Instance: mCODEGeneticSpecimenExample01 
+InstanceOf: GeneticSpecimen
+* id = "mCODEGeneticSpecimenExample01"
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/GeneticSpecimen"
+* status = #available "	Available"
+* subject = Reference(mCODEPatientExample01)
+* processing.timeDateTime = "2019-03-20"
+* collection.collector = Reference(mCODEPractitionerExample01) 
+* type = SPTY#TISS "Tissue"
+* collection.bodySite.coding = SCT#41224006 "Structure of lower lobe of left lung (body structure)"
```

Comment:
remove ".coding" -- bodySite is a CodeableConcept, in FSH it isn't necessary to be explicit